default-with-analytics/WSO2AM210-All-In-One-Deployment-VPC.template.yaml

```diff
@@ -599,43 +584,17 @@ Resources:
             - >-
               ln -s /mnt/efs/synapse-configs/synapse-configs
               /mnt/wso2am-2.1.0/repository/deployment/server/synapse-configs
+
             - apt-get --purge remove puppet mysql-client -y
             - rm -rf /etc/puppet
             - /mnt/wso2am-2.1.0/bin/wso2server.sh start
             - echo 'export HISTTIMEFORMAT="%F %T "' >> /etc/profile.d/history.sh
             - cat /dev/null > ~/.bash_history && history -c
```

Comment:
Same changes can be done here as proposed in the other launch configuratoin

Reply:
Agreed, will make the necessary changes as suggested in a future version as CloudFormation stack creation consumes a considerable amount of time.